HW13/PageObjects/DemoQA/Elements/ButtonsPage.cs

```diff
@@ -0,0 +1,33 @@
+﻿using HW13.Common.WebElements;
+using OpenQA.Selenium;
+
+namespace HW13.PageObjects.DemoQA.Elements
+{
+    public class ButtonsPage : BaseDemoQAPage
+    {
+        public const string doubleClickmeButtonMessage = "You have done a double click";
```

Comment:
1. Move constants to specific constant file
2. Constant variables must be like this doubleClickmeButtonMessage -> DoubleClickmeButtonMessage
3. Fix everywhere